### ISSUE TITLE
Support Multi Language

### ISSUE THREAD

**Issue by satotakumi** [jakerella/jquerySimplePassMeter#2]
I want to use this plugin in japanese.

Is it possible to support multiple languages ​​?

Thankyou.

**Comment by jakerella** [comment]
Yes, it can support multiple languages, but maybe not Japanese (or any non-Western character language). The issue is the [strength tests](https://github.com/jakerella/jquerySimplePassMeter/blob/master/jquery.simplePassMeter.js#L354-L398) which use a simple western [alphabet string](https://github.com/jakerella/jquerySimplePassMeter/blob/master/jquery.simplePassMeter.js#L348) for matching. Other than the password strength tests, all of the UI can be configured in the options through things like the [`defaultText`](https://github.com/jakerella/jquerySimplePassMeter/blob/master/jquery.simplePassMeter.js#L521), [`message` property](https://github.com/jakerella/jquerySimplePassMeter/blob/master/jquery.simplePassMeter.js#L534) for custom requirements, and [`ratings`](https://github.com/jakerella/jquerySimplePassMeter/blob/master/jquery.simplePassMeter.js#L571) UI.

So, you have two options:
- Fork the project and just change the strength tests to kanji (and anywhere else it needs to change)
- or submit a PR to make the strength test implementation modular with an option to override the default!

Sorry, but I don't have time to develop this right now. :(
